### ISSUE TITLE
Add makepot to composr's bin

### ISSUE THREAD

**Issue by mAAdhaTTah** [websharks/wp-i18n-tools#3]
This would be a useful enhancement:

https://getcomposer.org/doc/articles/vendor-binaries.md

**Comment by jaswsinc** [comment]
Thank you. So the suggestion is to create a binary that can be easily called upon; i.e., by editing the `composer.json` file in the `wp-i18n-tools` repo.

**Comment by mAAdhaTTah** [comment]
Yeah. You might not need to create a binary specifically, if `makepot.php` works as a shell script on its own. You just need to add it to `composer.json` so its gets added to the `vendor/bin` folder.
